package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
```

Comment:
We use single quotes in PHP and JavaScript in WordPress, so we should do the same in CSS (for consistency).